recipe/meta.yaml

```diff
@@ -62,6 +67,9 @@ test:
     - if exist %LIBRARY_PREFIX%\bin\warpx.2d.NOMPI.NOACC.DP.OPMD.PSATD.QED.exe (exit 0) else (exit 1)  # [win]
     - if exist %LIBRARY_PREFIX%\bin\warpx.3d.NOMPI.NOACC.DP.OPMD.PSATD.QED.exe (exit 0) else (exit 1)  # [win]
     - if exist %LIBRARY_PREFIX%\bin\warpx.RZ.NOMPI.NOACC.DP.OPMD.PSATD.QED.exe (exit 0) else (exit 1)  # [win]
+    # smoke tests: execute
+    - ./test.sh      # [unix and build_platform == target_platform]
+    - call test.bat  # [win]
```

Comment:
@isuruf do you know why this call is not run? I cannot spot it in `azure` output for the Windows builds :thinking: